### ISSUE TITLE
No error if missing default constructor

### ISSUE THREAD

**Issue by vincentnock** [johncarl81/parceler#253]
I've just experienced some annoying behavior that I'd like to share.

I was unable to build my project because one of my class annotated with `@Parcel` lacked the default constructor.
As I added an other constructor without noticing the `@Parcel`, it took me few hours to get what was exactly wrong.

The problem is that there was absolutely no error message about the problem.
Only the classic:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':debug'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

I think it's important to note that I use Android Annotations too.
I don't know if something can be made on the parceler side, but I prefer to put the issue here just in case.

**Comment by johncarl81** [comment]
Hmm, can you share code that caused this problem?

**Comment by vincentnock** [comment]
Ok, I figured out what the problem was. Nothing to do with Parceler !

I was tricked by the javac limitation of 100 errors show.
And as the Parceler error prevents Android Annotations to generate the _ classes, all the imports of thoses classes in my code show as errors, and the 100 limit is quickly reached.

I added this in by build.gralde, and the error did actually appear:
```groovy
gradle.projectsEvaluated {
    tasks.withType(JavaCompile) {
        options.compilerArgs << "-Xmaxerrs" << 500
    }
}
```
